### PR TITLE
Fix broken styles caused by antd update

### PR DESCRIPTION
**Pull request by shhdgit**
Prev menu ✅:
image

Current menu ❌:
image

Prev debug api collapse ✅:
image

Current debug api collapse ❌:
image

### CODE REVIEW

**Comment by shhdgit** [comment]
/cc @baurine @breeswish 
PTAL, thanks!

File: ui/lib/apps/DebugAPI/apilist/ApiList.module.less [hunk 1]
```diff
@@ -1,5 +1,19 @@
-.collapse_panel:not(:last-child) {
-  border-bottom: 1px solid #eee;
+.collapse_panel {
```
Comment (breeswish) [inline ui/lib/apps/DebugAPI/apilist/ApiList.module.less:1]: It looks pretty dangerous for us to rely on such styles. Our styles may be broken in future again when antd is upgraded.

**Review by breeswish — APPROVED** [review]

File: ui/lib/apps/DebugAPI/apilist/ApiList.module.less (hunk 1 above)
Comment (shhdgit) [reply ui/lib/apps/DebugAPI/apilist/ApiList.module.less:1]: Yes, we have a lot of hacking of antd style in our project.

Comment (breeswish) [reply ui/lib/apps/DebugAPI/apilist/ApiList.module.less:1]: Snapshot tests seems to be really helpful in these cases!

**Pull request merged**